File: src/main/java/com/application/lamion/repository/AppRepository.java
package com.application.lamion.repository;

import com.application.lamion.model.App;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface AppRepository extends JpaRepository<App, Long> {
    boolean existsByTitleAndUserId(String title, long userId);

    boolean existsByIdAndUserId(long id, long userId);

    long countByUserId(long userId);
}

File: server/src/main/java/com/application/lamion/repository/AppRepository.kt
package com.application.lamion.repository

import com.application.lamion.model.App
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface AppRepository : JpaRepository<App?, Long?> {
    fun existsByTitleAndUserId(title: String?, userId: Long): Boolean

    fun existsByIdAndUserId(id: Long, userId: Long): Boolean

    fun countByUserId(userId: Long): Long
}
